### ISSUE TITLE
Error building android app bundle 

### ISSUE THREAD

**Issue by rckrdstrgrd** [AdColony/AdColony-Android-SDK#72]
Cant build app using andorid app bundle (`./gradlew bundleRelease` ) when using  SDK version 4.1.3 (com.adcolony:sdk:4.1.3). If  I downgrade to AdColony 4.1.0 the build is successful.   

```
FAILURE: Build failed with an exception.
* What went wrong: 
Execution failed for task ':app:packageGoogleReleaseBundle'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
   > File 'root/AndroidManifest.xml' uses reserved file or directory name 'AndroidManifest.xml'.

```

**Comment by brittmullen** [comment]
Hey, thanks for reporting this - confirmed that I'm seeing the same issue locally. We'll look into what's changed build/configuration wise in 4.1.3 that could be causing this and correct it for our next release.

**Comment by brittmullen** [comment]
Fixed in new 4.1.4 release
